builder/common/run_config.go

```diff
@@ -390,11 +411,77 @@ type RunConfig struct {
 	// subnet-12345def, where Packer will launch the EC2 instance. This field is
 	// required if you are using an non-default VPC.
 	SubnetId string `mapstructure:"subnet_id" required:"false"`
-	// [Tenancy](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/dedicated-instance.html) used
-	// when Packer launches the EC2 instance, allowing it to be launched on dedicated hardware.
+	// The license configurations.
 	//
-	// The default is "default", meaning shared tenancy. Allowed values are "default",
-	// "dedicated" and "host".
+	// HCL2 example:
+	// ```hcl
+	// source "amazon-ebs" "basic-example" {
+	//   license_specifications {
+	//     license_configuration_request = {
+	//       license_configuration_arn = "${var.license_configuration_arn}"
+	//     }
+	//   }
+	// }
+	// ```
+	//
+	// JSON example:
+	// ```json
+	// "builders" [
+	//   {
+	//     "type": "amazon-ebs",
+	//     "license_specifications": [
+	//       {
+	//         "license_configuration_request": {
+	//           "license_configuration_arn": "{{user `license_configuration_arn`}}"
+	//         }
+	//       }
+	//     ]
+	//   }
+	// ]
+	// ```
+	//
+	//   Each `license_configuration_request` describes a license configuration,
+	//   the properties of which are:
+	//
+	//   - `license_configuration_arn` (string) - The Amazon Resource Name (ARN)
+	//     of the license configuration.
+	//
+	LicenseSpecifications []LicenseSpecification `mapstructure:"license_specifications" required:"false"`
+	// Describes the placement of an instance.
```

Comment:
@azr the [docs](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Placement.html) don't really go into much detail about the default placement. My interpretation is that these are scheduling hints that EC2 uses to filter the host onto which the instance is scheduled, otherwise EC2 chooses at random using black box heuristics that are unknown to/unknowable by the end-user.